server/src/main/resources/rules/rules.js

```diff
@@ -1043,7 +1043,7 @@ var CoverageCalculator = {
                 var supportedRoles = complianceTracker.enforces(prodAttr) ? complianceTracker.getAccumulatedValue(prodAttr) : [];
                 var consumerRole = consumer.role != null ? consumer.role : "";
                 if (!contains(supportedRoles, consumerRole)) {
-                    log.debug("  System addons not covered by: " + supportedAddOns);
+                    log.debug("  System addons not covered by: " + supportedRoles);
```

Comment:
Good catch. One thing to also change in the same line: the string of the debug statement should say 'roles' instead of 'addons'.